github/event_types.go

```diff
@@ -1248,3 +1248,63 @@ type WorkflowRunEvent struct {
 	Sender       *User         `json:"sender,omitempty"`
 	Installation *Installation `json:"installation,omitempty"`
 }
+
+type SecurityAdvisory struct {
+	GhsaID          *string                  `json:"ghsa_id,omitempty"`
```

Comment:
Just like `HTMLURL`, for idiomatic Go, I think we need this instead:

```suggestion
	GHSAID          *string                  `json:"ghsa_id,omitempty"`
```